Controller/MainController.py

```diff
@@ -85,7 +92,28 @@ def init_main_window(self):
 
         sub_layout.setContentsMargins(0, 0, 0, 0)
         sub_layout.addWidget(self.view_robot_state)
-        sub_layout.addWidget(self.view_field_screen)
+
+        # Ajout des boutons pour sauvegarder et restaurer la position des robots
+        field_widget = QWidget()
+        field_layout = QVBoxLayout()
+
+        buttons_widget = QWidget()
+        buttons_layout = QHBoxLayout(field_widget)
+
+        self.but_save_teams_formation = QPushButton("Save teams formation")
```

Comment:
I think the usual abbreviation for `button` is `btn` (I know, its nitpicking at this point 🌵 )